testData/solutions/web.json

```diff
@@ -234,5 +234,79 @@
                 }
             }
         ]
+    },
+    "de.fraunhofer.iao.C4A-TVM": {
+        "name": "C4A-TVM",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "web"
+                }
+            ]
+        },
+        "settingsHandlers": [
+            {
+                "type": "gpii.settingsHandlers.noSettings",
+                "capabilities": [
+                    "de\\.fraunhofer\\.iao\\.C4A-TVM.id",
```

Comment:
That appears to be a pattern but I couldn't find it documented at http://wiki.gpii.net/w/Solution_Registry or http://wiki.gpii.net/w/Settings_Handler. I'll change it in a minute.